profile.html

```diff
@@ -0,0 +1,249 @@
+					<div class="navbar-nav">
+						<a class="nav-link" aria-current="page" href="./home.html">Home</a>
+						<a class="nav-link active" href="./profile.html">Profile</a>
+					</div>
+				</div>
+				<div>
+					<button type="button" class="btn btn-primary px-3">
+						<i class="fa-regular fa fa-bell pe-2"></i>
+						5
+					</button>
+					<button
+						type="button"
+						class="navbar-toggler btn btn-primary px-3"
+						data-bs-toggle="collapse"
+						data-bs-target="#navbarSupportedContent"
+						aria-controls="navbarSupportedContent"
+						aria-expanded="false"
+						aria-label="Toggle navigation"
+					>
+						<i class="fa-solid fa fa-bars p-1 hamburger"></i>
+					</button>
+					<!-- <button type="button" class="btn btn-primary rounded-circle">
+						<i class="fa-solid fa fa-plus"></i>
+					</button> -->
+				</div>
+			</div>
+		</nav>
+		<main>
+			<section class="container">
+				<div
+					class="d-flex flex-column align-content-center justify-content-center mb-5"
+				>
+					<img
+						src="/assets/meg.jpeg"
+						alt="profile-picture of user"
+						class="rounded-circle align-self-center"
+						width="100"
+						height="100"
+					/>
+					<h1 class="align-self-center">Jon Doe</h1>
+				</div>
+
+				<div class="shadow rounded p-4 col">
+					<h2 class="mb-3">Online Friends</h2>
+					<div class="d-flex m-0">
+						<div class="position-relative m-2">
+							<div
+								class="position-relative"
+								style="
+									width: 50px;
+									height: 50px;
+									border-radius: 50%;
+									padding: 1px;
+								"
+							>
+								<img
+									src="/assets/me.png"
+									alt="..."
+									class="img-fluid rounded-circle"
+								/>
+							</div>
+							<div class="m-2 text-small">Jon</div>
+						</div>
+
+						<div class="position-relative m-2">
+							<div
+								class="position-relative"
+								style="
+									width: 50px;
+									height: 50px;
+									border-radius: 50%;
+									padding: 1px;
+								"
+							>
+								<img
+									src="/assets/me.png"
+									alt="..."
+									class="img-fluid rounded-circle"
+								/>
+							</div>
+							<div class="m-2 text-small">Jon</div>
+						</div>
+
+						<div class="position-relative m-2">
+							<div
+								class="position-relative"
+								style="
+									width: 50px;
+									height: 50px;
+									border-radius: 50%;
+									padding: 1px;
+								"
+							>
+								<img
+									src="/assets/me.png"
+									alt="..."
+									class="img-fluid rounded-circle"
+								/>
+							</div>
+							<div class="m-2 text-small">Jon</div>
+						</div>
+
+						<div class="position-relative m-2">
+							<div
+								class="position-relative"
+								style="
+									width: 50px;
+									height: 50px;
+									border-radius: 50%;
+									padding: 1px;
+								"
+							>
+								<img
+									src="/assets/me.png"
+									alt="..."
```

Comment:
```suggestion
									alt="User Avatar Image"
```

Alt attributes aren't just for fun. Non-visual users need them and they like to use the internet just as much as everyone else!